### ISSUE TITLE
Empty error messages when hovering over BPMN validation issues

### ISSUE THREAD

**Issue by Tupiet** [bpmn-io/bpmn-js-bpmnlint#69]
### Describe the Bug

TL;DR: While the modeler is showing that there are some errors in a BPMN, it doesn't show the text of the errors when hovering. 

I'm using the [modeler](https://github.com/bpmn-io/bpmn-js-examples) layout from bpmn-js-examples, which allows me to see and create BPMN files. Then, I've added bpmn-js-bpmnlint, because I need to see the errors that are in the diagram. I've followed the steps from the [bpmn-js-bpmnlint](https://github.com/bpmn-io/bpmn-js-bpmnlint), and the result is that, while it shows the correct number of errors, when I hover the error, the message is empty. 

### Steps to Reproduce

1. Clone the [modeler folder](https://github.com/bpmn-io/bpmn-js-examples) from bpmn-js-examples
2. Add bpmn-js-bpmnlint following the steps from the [bpmn-js-bpmnlint page](https://github.com/bpmn-io/bpmn-js-bpmnlint)
3. Add the bpmnlint-loader
4. Add the .bpmnlintrc
5. npm install
6. npm run dev
7. Open a BPMN file
8. Enable the BPMN validation (click the "pill" at the bottom-center)
9. Hover the errors
10. **See nothing in the error message**

I've created a repro, if this helps: [bpmn-editor](https://github.com/Tupiet/bpmn-editor). 

In the repro, I've commented with an asterisk (*) the lines in `src/app.js` and `webpack.config.js` I've added that were not in the modeler base, the lines I've added to add bpmnlint. 

With the repro, the steps are the following ones:

1. `git clone https://github.com/Tupiet/bpmn-editor.git`
2. `cd bpmn-editor`
3. `npm install`
4. `npm run dev`
5.  Open a BPMN file
7. Enable the BPMN validation (click the "pill" at the bottom-center)
8. Hover the errors
9. **See nothing in the error message**

<img width="329" alt="Captura de pantalla 2024-03-02 a las 13 03 49" src="https://github.com/bpmn-io/bpmn-js/assets/62516852/9853fc76-3076-43f0-b667-1a9fcb11352d">

### Expected Behavior

I would expect to see the text of the errors.

<img width="329" alt="Captura de pantalla 2024-03-02 a las 13 06 55" src="https://github.com/bpmn-io/bpmn-js/assets/62516852/3e95d802-cddd-4dfb-8cd8-725df4e883fa">

### Environment

 - Browser: [Safari 17.3, Chrome 121.0.x.x]
 - Library version: 
   - bpmn-js: 17.0.2
   - bpmn-js-bpmnlint: 0.22.2

Finally, thanks for this amazing library! Feel free to ask me for more details if you need it.

**Comment by nikku** [comment]
Thanks for opening this bug. I can reproduce it with your setup and transferred this issue to `bpmn-io/bpmn-js-bpmnlint`.

**Comment by barmac** [comment]
Notice however that if you run `npm start` in this repo, both error and warning tooltips display correctly. So it might be something about the user's setup.

**Comment by nikku** [comment]
@barmac I was about to comment exactly that. We'd still need to investigate what is up, as this affects a simple modification of the existing modeler example.

**Comment by Tupiet** [comment]
"Firstly, thank you for taking the time to review this. Strangely enough, in my case, doing `npm start` gives me the same result as `npm run dev`: error tooltips don't appear, while warnings do."

**Comment by barmac** [comment]
What browser do you use?

**Comment by Tupiet** [comment]
Tried both Safari and Chrome, same result.

**Comment by efim1703** [comment]
Did you manage to solve the problem?

**Comment by barmac** [comment]
I had a look again, and this is a clash of CSS between the library and user's code:

![image](https://github.com/user-attachments/assets/f60b025b-f53a-4f77-906a-4aab90db642b)

Class `.error` is used in the overlay but @Tupiet's repo makes it disappear via `display: none`.

**Comment by barmac** [comment]
So this is a non-issue in this library. Perhaps we should not use classes of such generic names, but I wouldn't call it a bug of the library.

**Comment by Tupiet** [comment]
Hi, thanks for taking the time for reviewing this. Unfortunately, the code @barmac mentions isn't mine, it's from the modeler example found in the [bpmn-io/bpmn-js-examples](https://github.com/bpmn-io/bpmn-js-examples) repo. Exactly, that CSS declaration is found [here](https://github.com/bpmn-io/bpmn-js-examples/blob/c88314b20e8c22714cb3517efcb6a54081ab396f/modeler/src/style.css#L54-L58). 

As it's explained in the first message of this issue, the code is just the modeler example + bpmn-js-bpmnlint. No custom code. Just that. 

If this helps, I've just tried to delete that CSS declaration, but that makes me stuck with the "Open a diagram" page. 

Thanks again for your help, and hope we can find a solution for this.

**Comment by barmac** [comment]
Thanks for the explanation. In that case, this is a problem with the example once you integrate bpmn-js-bpmlint with it.
I'd suggest you to check the example's code to make it use a different class name, or adjust the CSS to handle only the specific case. Maybe this change could be enough:

```diff
- .content:not(.with-error) .error
+ .content:not(.with-error) > .error
```

This should make the selector to hide only a direct child of `.content`, and not all elements with `.error` class.

**Comment by barmac** [comment]
I created a PR that should make integration easier: https://github.com/bpmn-io/bpmn-js-examples/pull/293

@Tupiet can you please integrate the changes in your repo, and give us a headsup in https://github.com/bpmn-io/bpmn-js-examples/pull/293 whether it helped?
